### PR TITLE
message to FE about changing network adapters

### PR DESCRIPTION
**Pull request by hitnik**
Background job implemented using APScheduler library. Initialized during the FastAPI startup by passing lifespan parameter.

### CODE REVIEW

File: framework/python/src/api/api.py
Comment (hitnik) [reply framework/python/src/api/api.py]: Done

File: framework/python/src/common/mqtt.py
Comment (hitnik) [reply framework/python/src/common/mqtt.py]: Done

**Comment by jboddey** [comment]
There's a few overlaps with PR #616. Will wait until that one is merged to make this PR easier to review.

**Review by sofyakurilova — APPROVED** [review]

**Pull request merged**
